CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+## 14.11.0
+
+### Minor Changes
+
+- Added: `ignoreAfterCombinators: []` to `selector-max-universal` ([#6275](https://github.com/stylelint/stylelint/pull/6275)) ([#6274](https://github.com/stylelint/stylelint/pull/6274)) ([@JounQin](https://github.com/JounQin)).
+
+### Patch Changes
+
+- Fixed: `selector-pseudo-class-allowed-list` end positions and message ([#6262](https://github.com/stylelint/stylelint/pull/6262)) ([#6274](https://github.com/stylelint/stylelint/pull/6274)) ([@JounQin](https://github.com/JounQin)).
```

Comment:
Will we have an opportunity to reorder the items in the changelog at some point during the workflow?

We've [consistently ordered](https://stylelint.io/maintainer-guide/pull-requests#merging) the items in our [changelog](https://stylelint.io/maintainer-guide/pull-requests#merging) since the beginning to make them easy to scan and consume by our users, e.g. wide-ranging fixes come first, specific rule fixes are alphabetically ordered etc.

Reply:
It seems not possible for now. Maybe I can PR to add some features for changesets.

Reply:
We can manually edit the changelog in this pull request just before merging to fix the order.

@ybiquitous Would you mind remembering to do that as part of the release?

Reply:
> Would you mind remembering to do that as part of the release?

Sure. 👍🏼